### DISCUSSION TITLE
How do I render my camera movement in the Nerf scene?

### DISCUSSION THREAD

**Discussion by vilewired** [NVlabs/instant-ngp#493] [Q&A]

**Comment by MIMNSI** [comment]
There's a tool behind the main GUI which lets you make camera path splines. 
Build the splines with it, save the file and then run this script to export the camera movement to mp4 
https://github.com/bycloudai/instant-ngp-Windows/blob/master/scripts/render.py

**Reply by thoi86** [reply]
@MIMNSI how do you run it? Can you copy paste the script to the insta-ngp folder? I tried installing the complete fork from bycloudai but the render.py script does not install to the dir.

**Reply by MIMNSI** [reply]
Yes, I copied the script into the instant-ngp folder and gave the appropriate paths in arguments.

**Reply by thoi86** [reply]
I am a bit new to this, but I copied the script into the scripts dir in instant-ngp folder and in anaconda I ran "python scripts/render.py --scene data/green_bush --n_seconds 30 --fps 60 --render_name teset --width 1920 --height 1080" but I keep getting this error:

File "C:\Users\thoib\Desktop\instant-ngp-Windows-master\instant-ngp\scripts\render.py", line 16, in
import pyngp as ngp # noqa
ModuleNotFoundError: No module named 'pyngp'

Do you have any idea what I am doing wrong?

**Reply by VibrantNebula** [reply]
Not sure if related but I see your quoatation mark isn't closed or in the right place. 

This is an example of one of my render command sequences in Anaconda. 

cd C:\Users\Admin\Desktop\NERF\instant-ngp
conda activate ngp

python "./scripts/render.py" --scene data/Zendoorway --n_seconds 30 --fps 30 --render_name Zendoorway_Final --width 1920 --height 1080

**Reply by VibrantNebula** [reply]
There's also this https://github.com/NVlabs/instant-ngp/issues/31#issuecomment-1014805317
